GNUmakefile

```diff
@@ -30,10 +29,6 @@ build: main.go pretest fmt
 install: main.go
 	$(GO) install -ldflags "$(LDFLAGS)"
 
-lint:
-	$(GO_OFF) get -u golang.org/x/lint/golint
```

Comment:
This line is removed, so GO_OFF definition at line 22 is not used anymore?

Reply:
https://github.com/vulsio/msfdb-list-updater/compare/27e9748a62f613067ba8efdc85c79dcc2641a4ed..379237245ea9dc07bd2a1d8755bc1e8344583f08